wpa_supplicant.conf

```diff
@@ -0,0 +1,4 @@
+ctrl_interface=DIR=/var/run/wpa_supplicant GROUP=netdev
+update_config=1
+country=US
```

Comment:
I need a way to make `country` configurable here. Maybe inside the ansible instead.